test/production/standalone-mode/required-server-files/required-server-files.test.ts

```diff
@@ -295,13 +295,13 @@ describe('required server files', () => {
       case: 'notFound no revalidate',
       path: '/optional-ssg/not-found-1',
       dest: '/somewhere',
-      cacheControl: 's-maxage=31536000, stale-while-revalidate',
+      cacheControl: 's-maxage=31536000',
```

Comment:
Why did we drop the `stale-while-revalidate` value here? That can break behavior when deployed as it won't revalidate in background anymore

Reply:
`stale-while-revalidate` without a time value is malformed. When revalidate is false, we just set `s-maxage` to the fixed maximum of one year. There is no revalidation expected for this page. I believe this change is correct.

Reply:
https://github.com/vercel/next.js/blob/canary/test/production/standalone-mode/required-server-files/pages/optional-ssg/%5B%5B...rest%5D%5D.js#L18

It's returning `notFound` with no revalidate value so it should be `false` which is the default so should be the full year